app/actions/definitions/documents.tsx

```diff
@@ -108,6 +110,63 @@ export const unstarDocument = createAction({
   },
 });
 
+export const subscribeDocument = createAction({
+  name: ({ t }) => t("Subscribe"),
+  section: DocumentSection,
+  // TODO: Use other icon.
+  icon: <BookmarkedIcon />,
+  keywords: "subscribe",
```

Comment:
```suggestion
```

Don't need keywords if they are just the same as the name, only if we have synonyms. In this case I can't really think of any…